lib/MarketService.php

```
@@ -85,6 +86,8 @@ public function installApp($appId) {
 			$package = $this->downloadPackage($appId);
 			$this->installPackage($package);
 			//Market is down or misfunctional
+		} catch (ClientException $e){
+			throw new AppManagerException('No marketplace connection');
```

can we not swallow the underlying exception here and in the other places:
```
throw new AppManagerException('No marketplace connection', 0, $e);
```

It is interesting to know if it was a Client or ServerException ... as well as the message they (shoud) contain